src/Microsoft.DotNet.Darc/src/DarcLib/VirtualMonoRepo/VmrPatchHandler.cs

```diff
@@ -234,8 +234,12 @@ public async Task ApplyPatch(SourceMapping mapping, VmrIngestionPatch patch, Can
         // We have to give git a relative path with forward slashes where to apply the patch
         var destPath = _vmrInfo.GetRepoSourcesPath(mapping)
             .Replace(_vmrInfo.VmrPath, null)
-            .Replace("\\", "/")
-            [1..];
+            .Replace("\\", "/");
+
+        if (destPath[0] == '/')
+        {
+            destPath = destPath.Substring(1);
+        }
```

Comment:
I think you can also do:

```suggestion
            .Replace("\\", "/")
            .TrimStart('/');
```